test/services/sync/strategies/CSVSyncer/CSVSyncer.integration.test.js

```diff
@@ -24,12 +32,10 @@ describe('CSVSyncer Integration', () => {
 	});
 
 	describe('Scenario 0 - Missing authentication', () => {
-		const SCHOOL_ID = '0000d186816abba584714c5f';
-
 		const scenarioParams = {
 			query: {
 				target: 'csv',
-				school: SCHOOL_ID,
+				school: '0000d186816abba584714c5f',
```

Comment:
If you like you can fetch the default schoolId from testObjects over testObjects.options.schoolId.

Reply:
thanks!